ui/app/abilities/recommendation.js

```diff
@@ -3,7 +3,7 @@ import { computed } from '@ember/object';
 import { or } from '@ember/object/computed';
 
 export default class Recommendation extends AbstractAbility {
-  @or('bypassAuthorization', 'selfTokenIsManagement', 'policiesSupportAcceptingOnAnyNamespace')
+  @or('selfTokenIsManagement', 'policiesSupportAcceptingOnAnyNamespace')
```

Comment:
Can you not use recommendations without ACLs?

Reply:
hmm… you’re right that this is not a correct way to determine this! 😢

I’ve now added 24c3a95 which introduces a license-fetching step when loading the application akin to when the policies are fetched. On enterprise `GET /v1/operator/license` returns something that we can use to check for whether a feature exists:

```
{
  …,
  License: {
    …,
    Features: […, 'Dynamic Application Sizing', …],
    …,
  …,
}
```

On OSS the endpoint `501`s so I return an empty license with no features instead.

Reply:
🤩🤩🤩🤩

This is fantastic. I'm sorry this bug ended up being bigger than it appeared, but having this level of feature-checking is an excellent addition.

Reply:
I’m glad to have finally added this despite the sadness of another spurious failure message:

<img width="479" alt="image" src="https://user-images.githubusercontent.com/43280/98294273-4c7fa300-1f75-11eb-97f1-4b817f8a633e.png">

Reply:
😳